docs/_layouts/tutorials.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial_index.html
+---
+<!DOCTYPE html>
+<html lang="en-us">
+
+  {% include head.html %}
+
+  <body>
+  <!-- Google Tag Manager (noscript) -->
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5DGS7TG"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+  <!-- End Google Tag Manager (noscript) -->
+
+    {% include top_nav.html %}
+
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <h1 class="page-title" >{{ page.title }}</h1>
+          <div class="page-subtitle">
+            <p class="page-subtitle-left">Estimated time to complete: {{ page.time-complete }} min</p>
+            <p class="page-subtitle-right">Products used: {{ page.products-used }}</p>
+          </div>
+
+          <div class="row page-wrapper">
+            <!-- left-side of tutorial with content -->
+            <div class="col-xs-12 col-s-12 col-md-3 tutorial-left">
+              <div class="progressbar-index">
+                <ul class="progressbar">
+
+                  {% for item in site.data.navigation.tutorial-tiles %}
```

Comment:
should `tutorial-tiles` here be a parameter (not sure if that's possible, but I'm shooting for the moon!) so that multiple tutorials could use this layout, and this could reference the layout for any given tutorial?

Reply:
I tried a couple of ways to accomplish this (collections, passing parameters with assign from specific tile and assign content in tutorial layout) but it seems that somehow it is being converted into a string (I think).
@jonahx @typaulhus do you know another way to pass a parameter into a for loop with jekyll?

Reply:
I really like this idea! 
I realize that this is taking longer than expected for implementation so I have logged as issue #680  and when we start to think about adding more tutorials, we can revisit this one